### PR TITLE
Emit action selector id

### PR DESCRIPTION
**Pull request by kamleshbhalui**
Emit action selector and profile id same as in bfrt

### CODE REVIEW

**Comment by usha1830** [comment]
Please add  context json for atleast one testcase. Insert it as a text file.

**Comment by kamleshbhalui** [comment]
[contextid.json.txt](https://github.com/kamleshbhalui/p4c/files/10442826/contextid.json.txt)

**Comment by kamleshbhalui** [comment]
[p4info.txt](https://github.com/kamleshbhalui/p4c/files/10443148/p4info.txt)

**Comment by kamleshbhalui** [comment]
[bfrt.json.txt](https://github.com/kamleshbhalui/p4c/files/10443158/bfrt.json.txt)

File: backends/dpdk/dpdkContext.cpp [hunk 1]
```diff
@@ -210,6 +210,9 @@ size_t DpdkContextGenerator::getHandleId(cstring name) {
         if (x.first.find(name.c_str())) {
             id = context_handle_map[x.first];
             break;
+        } else if (name.find(x.first.c_str()) && name.endsWith("_sel")) {
```
Comment (usha1830) [inline backends/dpdk/dpdkContext.cpp:213]: We are creating new ID here. This leaves different Selector table ids in context json and bfrt.json.

**Comment by kamleshbhalui** [comment]
[contextid1.json.txt](https://github.com/kamleshbhalui/p4c/files/10444687/contextid1.json.txt)
attaching new contextid1.json

**Review by usha1830 — APPROVED** [review]
Looks fine

**Review by psivanup — APPROVED** [review]
Looks fine

**Pull request closed without merging**